src/views/Pathways/Milestones/Edit/index.js

```diff
@@ -8,7 +8,7 @@ import { withTheme } from '@material-ui/core';
 import { selectors, setMilestoneToView, addOrEditMilestone } from '../../store';
```

Comment:
This file can easily be named just Milestones/Edit.js. Similar to what I pointed out with Add.js.